consensus/state_processing/src/per_epoch_processing/altair/participation_flag_updates.rs

```diff
@@ -0,0 +1,21 @@
+use crate::EpochProcessingError;
+use core::result::Result;
+use core::result::Result::Ok;
+use types::beacon_state::BeaconState;
+use types::eth_spec::EthSpec;
+use types::participation_flags::ParticipationFlags;
+use types::VariableList;
+
+//TODO: there's no EF test for this one
+pub fn process_participation_flag_updates<T: EthSpec>(
+    state: &mut BeaconState<T>,
+) -> Result<(), EpochProcessingError> {
+    *state.previous_epoch_participation_mut()? =
+        std::mem::take(state.current_epoch_participation_mut()?);
+    *state.current_epoch_participation_mut()? = VariableList::new(vec![
+        ParticipationFlags::default(
+        );
```

Comment:
Hmm, looks like a bug in `rustfmt`. No change required, just pointing it out. I tried to put the `);` back onto L16 but `rustfmt` put it back :shrug:

Reply:
How weird!